<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OutputDataOptions</class>
 <widget class="QWidget" name="OutputDataOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>474</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Output Data Options</string>
     </property>
     <layout class="QGridLayout" name="outputDataOptionsLayout">
      <item row="6" column="1">
       <widget class="QCheckBox" name="mCheckBoxOrientation">
        <property name="text">
         <string>Orientation (XYZ)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="mCheckBoxVel">
        <property name="text">
         <string>Velocity</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="mCheckBoxSpeed">
        <property name="text">
         <string>Speed</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QCheckBox" name="mCheckBoxGR">
        <property name="text">
         <string>Ground Rate</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QCheckBox" name="mCheckBoxRoll">
        <property name="text">
         <string>Roll</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="mCheckBoxAltMSL">
        <property name="text">
         <string>Altitude - MSL</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBox" name="mCheckBoxElev">
        <property name="text">
         <string>Ground Elevation</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QCheckBox" name="mCheckBoxPitch">
        <property name="text">
         <string>Pitch</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QCheckBox" name="mCheckBoxHeading">
        <property name="text">
         <string>Heading</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="mCheckBoxLatLon">
        <property name="text">
         <string>Latitude/Longitude</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="mCheckBoxPos">
        <property name="text">
         <string>Position (XYZ)</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QCheckBox" name="mCheckBoxAR">
        <property name="text">
         <string>Altitude Rate</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="mCheckBoxAltAGL">
        <property name="text">
         <string>Altitude - AGL</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QCheckBox" name="mCheckBoxAFV">
        <property name="text">
         <string>Acceleration From Velocity</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QCheckBox" name="mCheckBoxAFP">
        <property name="text">
         <string>Acceleration From Position</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QCheckBox" name="mCheckBoxFC">
        <property name="text">
         <string>Fuel Consumption (lbm/hr)</string>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QCheckBox" name="mCheckBoxFR">
        <property name="text">
         <string>Fuel Remaining (lbm/hr)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
